word2vec.py

```diff
@@ -116,45 +124,39 @@ def get_scores(model, pairs):
                 "score": score
             }
             similarity_scores_df = similarity_scores_df.append(new_row, ignore_index=True)
-    similarity_scores_df.to_csv("similarity_scores.tsv", sep="\t")
+
+    similarity_scores_df.to_csv("outputs/similarity_scores.tsv", sep="\t", index=False)
 
 
-def calc_roc(scores_filename):
+def dummy(scores_filename):
     """
-    Generates ROC curve.
-    @param scores_filename: file containg similarity scores
+    Runs dummy classifier. Outputs dummy_scores.tsv.
+    @param scores_filename: file containing gene-disease associations and class 
+        (also contains word2vec similarity scores)
     """
-    df = pd.read_csv(scores_filename, sep="\t")
-    labels = np.array(df[["class"]].values.tolist())
-    predictions = np.array(df[["score"]].values.tolist())
-    return(roc_curve(labels, predictions))
+    scores_df = pd.read_csv(scores_filename, sep="\t")
+    X = scores_df.drop(["class", "score"], 1)
+    y = scores_df["class"]
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3, random_state=0)
```

Comment:
```suggestion
    (
      X_train, X_test, 
      y_train, y_test
     ) = train_test_split(
           X, y, 
           test_size=0.3, 
           random_state=0
        )
```

Reply:
Will need to reindent for python to work, but the whitespace makes it easier to read.